build.gradle

```diff
@@ -0,0 +1,212 @@
+buildscript {
```

Comment:
I would also prefer one root build.gradle file and one file for each sub project. It the more common and cleaner solution.